src/main/scala/com/itechart/ny_accidents/utils/StringUtils.scala

```diff
@@ -11,4 +12,16 @@ object StringUtils {
     }
   }
 
+  def getLineMatchPercentage(firstString: String, secondString: String): Double = {
```

Comment:
I'd like to see a couple of unit test for this tricky method. Could you please add a few? Thanks

Reply:
I'm not sure about usability of this method, but tests were written